promscale/installation/promscale-with-timescale-cloud/kubernetes.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Kubernetes
+excerpt: Install Promscale Connector on a Kubernetes cluster with Timescale Cloud
+product: promscale
+keywords: [Kubernetes, analytics, Helm]
+tags: [install]
+related_pages:
+  - /promscale/:currentVersion:/guides/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleInstallPrerequisite from 'versionContent/_partials/_promscale-install-pre-requisite.mdx';
+import PromscaleSendData from 'versionContent/_partials/_promscale-send-data.mdx';
+
+# Install Promscale on Kubernetes
+
+You can install Promscale on Kubernetes using Helm or using a manifest file.
+
+<PromscaleInstallPrerequisite />
+
+## Before you begin
+
+*   Install Helm. For more information, including packages and installation
+    instructions, see the [Helm documentation][install-helm].
+*   Create a [TimescaleDB service][create-service] on Timescale Cloud.
+
+### Install the Promscale using Helm chart
+
+Promscale needs to access your TimescaleDB database. You
+can provide the database URI, or specify connection parameters.
+
+<procedure>
+
+#### Installing the Promscale using Helm chart
+
+1.  Add the TimescaleDB Helm chart repository:
+
+    ```bash
+    helm repo add timescale 'https://charts.timescale.com'
+    ```
```

Comment:
@VineethReddy02  do you need this?

Reply:
Yes, that's basically adding the timescale helm registry.